.github/workflows/tf-check-secure-estimate.yml

```diff
@@ -10,8 +10,6 @@ permissions:
   pull-requests: write
 
 env:
-  AWS_ACCESS_KEY_ID : "${{ secrets.AWS_ACCESS_KEY_ID }}"
```

Comment:
Should be declared to work with the check code step

Reply:
As discussed, added `-backend=false`, which should fix the check issue